File: lib/Model/UserModel.dart
class UserModel {
  String? name;
  String? emailAddress;
  String? rollNumber;
  String? password;
  String? imageUrl;

  UserModel({
    this.name,
    this.emailAddress,
    this.rollNumber,
    this.password,
    this.imageUrl,
  });

  UserModel.fromJson(Map<String, dynamic> json) {
    name = json['name'];
    emailAddress = json['emailAddress'];
    rollNumber = json['rollNumber'];
    password = json['password'];
    imageUrl = json['imageUrl'];
    ;
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['name'] = this.name;
    data['emailAddress'] = this.emailAddress;
    data['rollNumber'] = this.rollNumber;
    data['password'] = this.password;
    data['imageUrl'] = this.imageUrl;

    return data;
  }
}

File: Server/Modules/UserModel.js
const mongoose = require("mongoose");

const UserModule = mongoose.Schema({
  name: {
    require: true,
    type: String,
  },
  emailAddress: {
    require: true,
    type: String,
  },

  rollNumber: {
    require: true,
    type: String,
  },
  password: {
    require: true,
    type: String,
  },
  imageUrl: {
    require: true,
    type: String,
  },
});
const userModule = mongoose.model("userModule", UserModule);
module.exports = { userModule, UserModule };
